plugins/axon/skills/knowledge-base/SKILL.md

```
@@ -0,0 +1,85 @@
+
+Choose the Axon surface by source shape:
+
+| Need | Axon surface |
+|---|---|
+| Discover URLs only | `axon map <url>` |
+| Capture a docs site or section as artifacts | `axon crawl <url> --output-dir <dir>` |
+| Fetch selected pages | `axon scrape <url> --output-dir <dir>` |
+| Discover topic sources | `axon search` or `axon research` |
+| Ingest repos, RSS, Reddit, YouTube, or sessions | `axon ingest` / `axon sessions` |
+| Refresh existing indexed corpus | `axon refresh [filter] --yes` |
+| Schedule freshness | `axon crawl|scrape|embed|ingest ... --fresh <Nd>` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)